src/passwd.c

```diff
@@ -682,11 +697,11 @@ static void update_shadow (void)
  *	-l	lock the password of the named account (*)
  *	-n #	set sp_min to # days (*)
  *	-r #	change password in # repository
- *	-s	execute chsh command to interpret flags
```

Comment:
What heppened to -s?

Reply:
Please don't remove that line.

Reply:
It seems that -s was never there or it is long gone. I hesitated if I can usurp -s short flag for --stdin. Now another PR has been merged that cleaned this comment so I updated my PR to
- not remove old comment
- use -s as a short option for --stdin

Reply:
Ahh, makes sense.  I checked, and none of FreeBSD, NetBSD, nor OpenBSD use -s, so that LGTM.